website/docs/cloud-docs/workspaces/settings/index.mdx

```diff
@@ -46,6 +44,10 @@ The display name of the workspace.
 
 ~> **Important:** Since some API calls refer to a workspace by its name, changing the name can sometimes break existing integrations.
 
+### Description (Optional)
+
+You can enter an optional description for the workspace.
```

Comment:
What about something like this? I'm wondering if we could provide value beyond just restating the name of the field. 

```suggestion
Enter a brief description of the workspace's purpose or types of infrastructure.
```

Reply:
The wording works. But it is an option description. So should we put {Optional) at the start?

Reply:
hmm i don't have strong feelings on that. It's in the title, which is why I left it out, but you're right - maybe we should add to the text as well! feel free to update my suggestion :)